pom.xml

```diff
@@ -97,38 +136,16 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>6.0.0</version>
             <scope>test</scope>
         </dependency>
 
         <!-- libs necessary for tests -->
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>11.0.1</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
-            <version>4.5</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>2.21.0</version>
             <scope>test</scope>
         </dependency>
 
-        <!-- lib needed to support legacy git plugin -->
-        <dependency>
-            <groupId>org.spearce</groupId>
-            <artifactId>jgit</artifactId>
```

Comment:
Deleting stuff that seemed to be there to support some truly ancient version of the `git` plugin.